conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -358,9 +335,12 @@ private void updateSession() {
     }
 
     private void createContentMetadata() {
-        SourceItem sourceItem = bitmovinPlayer.getConfig().getSourceItem();
-        if (sourceItem != null) {
-            contentMetadataBuilder.setAssetName(sourceItem.getTitle());
+        Source source = bitmovinPlayer.getSource();
+        if (source != null) {
+            SourceConfig sourceConfig = source.getConfig();
+            if (sourceConfig != null) {
+                contentMetadataBuilder.setAssetName(sourceConfig.getTitle());
+            }
```

Comment:
nit: you could use early return here to keep the happy path not indented.

Reply:
The control is not returning if source or sourceConfig is null so cannot return early return here.

Reply:
```Java
Source source = bitmovinPlayer.getSource();
if (source == null) return;

SourceConfig sourceConfig = source.getConfig();
if (sourceConfig == null) return;

...
```

Should work and improves readability imo.

Reply:
The original code does not return if source of sourceConfig is null

Reply:
I know, you don't have to return an instance or primitive, just break the execution order with `return;`.